drools-scenario-simulation/drools-scenario-simulation-backend/src/test/java/org/drools/scenariosimulation/backend/expression/DMNFeelExpressionEvaluatorTest.java

```diff
@@ -289,4 +290,10 @@ public void isStructuredInput() {
         assertFalse(expressionEvaluator.isStructuredInput(Integer.class.getCanonicalName()));
         assertFalse(expressionEvaluator.isStructuredInput(String.class.getCanonicalName()));
     }
+    
+    @Test
+    public void drools6337() {
+        assertTrue(expressionEvaluator.evaluateUnaryExpression("now() > ?", LocalDateTime.of(2020, 9,6,11,34), LocalDateTime.class).isSuccessful());
+        assertTrue(expressionEvaluator.evaluateUnaryExpression("today() > ?", LocalDate.of(2020, 9, 6), LocalDateTime.class).isSuccessful());
```

Comment:
It will be never probably an issue but what about removing hardcoded values?
```suggestion
        LocalDateTime now = expressionEvaluator.evaluateLiteralExpression("now()", LocalDateTime.class.getCanonicalName());
        LocalDate today = expressionEvaluator.evaluateLiteralExpression("today()", LocalDate.class.getCanonicalName());
        assertNotNull(now);
        assertNotNull(today);
        assertTrue(expressionEvaluator.evaluateUnaryExpression("now() > ?", now.minusDays(1), LocalDateTime.class).isSuccessful());
        assertTrue(expressionEvaluator.evaluateUnaryExpression("today() > ?", today.minusDays(1), LocalDateTime.class).isSuccessful());
```